src/components/molecules/ZopaFooter/ZopaFooter.test.tsx

```diff
@@ -17,18 +17,9 @@ describe('<ZopaFooter />', () => {
     date.finishMocking();
   });
 
-  it('renders the component with default props', async () => {
-    const { container } = render(<ZopaFooter />);
-    expect(container.firstChild).toMatchSnapshot();
-  });
-
-  it('renders the component with a legalOnly prop', async () => {
-    const { container } = render(<ZopaFooter legalOnly />);
-    expect(container.firstChild).toMatchSnapshot();
-  });
-
   it('renders the component with no a11y violations', async () => {
     const { container } = render(<ZopaFooter />);
+    expect(container.firstChild).toMatchSnapshot();
     const results = await axe(container.innerHTML);
     expect(results).toHaveNoViolations();
   });
```

Comment:
I know there weren't tests for 🤫 , could we take the opportunity to add for the `baseUrl` prop? 🙏🏻
( _no need to verify every link, verifying the couple of them render with it should be enough_ )

Reply:
test added! :)